src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -9,6 +9,8 @@ void npe(String a, String b) {
   } else {
 
   }
+  if (a == null)
+   a.charAt(123);
```

Comment:
**Reporter**: FINDBUGS
**Rule**: NP_LOAD_OF_KNOWN_NULL_VALUE
**Severity**: INFO
**File**: se/bjurr/violations/lib/example/MyClass.java L13
**Source**: se.bjurr.violations.lib.example.MyClass

Load of known null value&#10;&#10;&#10;  &lt;p&gt; The variable referenced at this point is known to be null due to an earlier&#10;   check against null. Although this is valid, it might be a mistake (perhaps you&#10;intended to refer to a different variable, or perhaps the earlier check to see if the&#10;variable is null should have been a check to see if it was non-null).&#10;&lt;/p&gt;&#10;


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1613869392>*